app.py

```diff
@@ -4,9 +4,73 @@
 app = Flask(__name__)
 app.config.from_object('flask_config.Config')
 
-@app.route('/')
+@app.route('/') #redirect root to the url or route mapped to the index function
+def root():
+    return redirect(url_for('index'))
+
+@app.route('/items', methods = ["GET", "POST", "PUT"])
 def index():
-    return 'Hello World!'
+    if request.method == 'POST':
+        new_item = session.add_item(request.form['Title'])  ## Title is the name used in new.html
+##      session._DEFAULT_ITEMS.append(new_item) #append to list instead calling session
+        return redirect(url_for('index'))
+### tried using PUT doesnt save input from form :(
```

Comment:
Yes forms can only send GET or POST requests (without leveraging javascript)

For this exercise you will only need to rely on POST (so don't worry about the PUT case here)